en/flight_controller/spracingh7extreme.md

```diff
@@ -0,0 +1,70 @@
+# SP Racing H7 EXTREME (PX4 Edition)
+
+The SP Racing H7 EXTREME is a feature packed FC/PDB with DUAL ICM20602 gyros, H7 400Mhz CPU*, high-precision BMP388 barometer, SD Card socket, current sensor, 8 easily accessible motor outputs, OSD, Microphone, Audio output, and more.
+
+It can be used easily for small to large quads but also for planes, octocoptors and more advanced frames.  It's best used with separate ESCs as it features a built-in power distribution board (PDB), but wiring up a 4in1 ESC is easy too.
+
+There is also a 12 pin stacking connector which provides 4 more motor outputs, SPI and a UART connectivity. 
+
+![SPRacingH7EXTREME PCB Top](../../assets/flight_controller/spracingh7extreme/spracingh7extreme-top.jpg)
+
+![SPRacingH7EXTREME PCB Bottom](../../assets/flight_controller/spracingh7extreme/spracingh7extreme-bottom.jpg)
+
+
+## Key Features

```

Comment:
Would it be possible to divide these into subcategories - eg. like in http://docs.px4.io/master/en/flight_controller/pixhawk4_mini.html#quick-summary - sensors, interfaces, etc. That makes it a little easier to parse than this very large list, and compare to similar products.

-

Reply:
done

Reply:
Done